src/app/globals.css

```
@@ -7,9 +7,16 @@
 	--color-green-dark: #1f3327;
 	--color-ink: #26241f;
 	--font-display: var(--font-cormorant), Georgia, serif;
+	--font-accent: var(--font-cormorant), Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)